doc/10-icinga-template-library.md

```diff
@@ -2799,6 +2799,30 @@ smart_attributes_config_path    | **Required.** Path to the smart attributes con
 smart_attributes_device         | **Required.** Device name (e.g. /dev/sda) to monitor.
 
 
+#### smart-advanced <a id="plugin-contrib-command-smart-advanced"></a>
+
+The [check_smart](https://www.claudiokuenzler.com/monitoring-plugins/check_smart.php) plugin uses the `smartctl` command from the `smartmontools` package to monitor SMART values of physical drives. Hard, solid state and NVMe drives are supported. Both Direct drive access or drive behind a storage controller (such as RAID) are supported.
+
+Either smart_device (-d) or smart_global (-g) is required. smart_interface (-i) is always required.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                   | Description
+-----------------------|-----------------------------------------------------------------------
+smart_device           | **Required.** Path to physical block device to be SMART monitored, e.g. /dev/sda.
+smart_interface        | **Required.** Drive's interface type, must be one of: auto, ata, scsi, nvme, 3ware,N, areca,N, hpt,L/M/N, cciss,N, megaraid,N.
+smart_global           | **(Required.)** A glob expression of physical devices to be monitored, eg -g "/dev/sd[a-z]".
+smart_raw_list         | **Optional.** List (comma separated, without spaces!) of SMART attributes to check for their raw values.
+smart_exclude_list     | **Optional.** List of (comma separated) SMART attributes which should be excluded (=ignored) from checks, but still appear in perfdata.
+smart_exclude_all_list | **Optional.** List of (comma separated) SMART attributes which should be excluded (=ignored) **completely** from checks and perfdata.
+smart_bad              | **Optional.** Threshold value (integer) when to warn for N bad entries (ATA: Current Pending Sector, SCSI: Grown defect list). Note: Deprecated for ATA drives, use `smart_warn` instead. Continue to use this for SCSI drives.
+smart_warn             | **Optional.** Comma separated list of thresholds for ATA drives (e.g. `'Reallocated_Sector_Ct=10,Current_Pending_Sector=62'`).
+smart_selftest         | **Optional.** If set to true, additionally check SMART's selftest log for errors.
+smart_quiet            | **Optional.** If set to true, only show failing drive(s) when faults are detected (only affects output when used with `smart_global`).
+
+Either smart_device (-d) or smart_global (-g) is required. smart_interface (-i) is always required.
```

Comment:
This is the same as above the table.

Reply:
Better twice than never ^^. Will keep the note on top.